### PR TITLE
Add support for Python 2

### PR DESCRIPTION
**Pull request by fmassa**
This PR adds support for Python 2, and should fix https://github.com/facebookresearch/maskrcnn-benchmark/issues/7

I've successfully run `tools/train.py` and `tools/test.py` after these fixes, and training / inference succeeded without errors.

I'll run a quick schedule train/test cycle to verify that the numbers match the expected results.

### CODE REVIEW

File: maskrcnn_benchmark/modeling/poolers.py [hunk 1]
```diff
@@ -72,8 +72,8 @@ def __init__(self, output_size, scales, sampling_ratio):
         self.output_size = output_size
         # get the levels in the feature map by leveraging the fact that the network always
         # downsamples by a factor of 2 at each level.
-        lvl_min = -math.log2(scales[0])
-        lvl_max = -math.log2(scales[-1])
+        lvl_min = -torch.log2(torch.tensor(scales[0], dtype=torch.float32)).item()
```
Comment (rbgirshick) [inline maskrcnn_benchmark/modeling/poolers.py]: `import math` can be removed now.

**Comment by fmassa** [comment]
Thanks for the review @rbgirshick !

I'm performing some full trainings to verify that accuracy is the same.
Training on the reduced scheduled gave expected results, as well as inference on pre-trained models.
Once I verify that this works as expected on the full training I'll merge it.

**Comment by fmassa** [comment]
Here are the results for training on Mask R-CNN R-50 FPN 1x. Accuracies match the baselines, and it seems that the Python 2 runtime is a bit slower. I've run it twice.

backbone | type | lr sched | im / gpu | train mem(GB) | train time (s/iter) | total train time(hr) | inference time(s/im) | box AP | mask AP | model id
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
R-50-FPN | Mask | 1x | 2 | 5.2 | 0.4657 | 11.6 | 0.13757 + 0.034 | 37.7 | 34.3 | 6486235
R-50-FPN | Mask | 1x | 2 | 5.2 | 0.4670 | 11.6 | 0.14075 + 0.034 | 37.6 | 34.2 | 6486657

I'm merging this.

File: maskrcnn_benchmark/modeling/poolers.py (hunk 1 above)
Comment (fmassa) [reply maskrcnn_benchmark/modeling/poolers.py]: Done, thanks!
